VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs

```diff
@@ -404,6 +408,77 @@ await Clients.Group(judge.Username.ToLowerInvariant())
+                    .Where(p => participant.LinkedParticipants.Select(x => x.LinkedId)
+                        .Contains(p.Id)
+                    ).ToList();
+
+                await Clients.Group(participant.Username.ToLowerInvariant())
+                    .ParticipantRemoteMuteMessage(participantId, conferenceId, isRemoteMuted);
+                _logger.LogTrace(
+                    "Participant remote mute status updated: Participant Id: {ParticipantId} | Conference Id: {ConferenceId} to {IsRemoteMuted}",
+                    participantId, conferenceId, isRemoteMuted);
+                Task.WaitAll(
+                    linkedParticipants.Select(linkedParticipant => Clients
+                        .Group(linkedParticipant.Username.ToLowerInvariant())
+                        .ParticipantRemoteMuteMessage(participantId, conferenceId, isRemoteMuted)).ToArray());
+
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(ex,
+                    "Error occured when updating participant {ParticipantId} in conference {ConferenceId} remote mute status to {IsRemoteMuted}",
+                    participantId, conferenceId, isRemoteMuted);
+            }
+        }
+
+        /// <summary>
+        /// Publish a participant's hand status (i.e. raised or lowered)
+        /// </summary>
+        /// <returns></returns>
+        public async Task UpdateParticipantHandStatus(Guid conferenceId, Guid participantId, bool isRaised)
+        {
+            try
+            {
+                var conference = await GetConference(conferenceId);
+                var participant = conference.Participants.Single(x => x.Id == participantId);
+                var linkedParticipants = conference.Participants
+                    .Where(p => participant.LinkedParticipants.Select(x => x.LinkedId)
+                        .Contains(p.Id)
+                    ).ToList();
+
+                var judge = conference.Participants.Single(x => x.IsJudge());
+                await Clients.Group(judge.Username.ToLowerInvariant())
+                    .ParticipantHandRaiseMessage(participantId, conferenceId, isRaised);
+                await Clients.Group(participant.Username.ToLowerInvariant())
+                    .ParticipantHandRaiseMessage(participantId, conferenceId, isRaised);
+                Task.WaitAll(
+                    linkedParticipants.Select(linkedParticipant => Clients
+                        .Group(linkedParticipant.Username.ToLowerInvariant())
+                        .ParticipantHandRaiseMessage(participantId, conferenceId, isRaised)).ToArray());
+                _logger.LogTrace(
+                    "Participant hand status updated: Participant Id: {ParticipantId} | Conference Id: {ConferenceId} to {IsHandRaised}",
+                    participantId, conferenceId, isRaised);
+
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(ex,
```

Comment:
i dont agree with catching just to add logging but i see its done for the other methods so letting it slide

Reply:
I think we've had problems int the past when an unhandled exception kills the connection